pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.towncrier]
+package = "beetmoverscript"
+package_dir = "."
+filename = "CHANGELOG.rst"
+issue_format = "`#{issue} <https://github.com/mozilla-releng/beetmoverscript/issues/{issue}>`_"
+title_format = "[{version}] = ({project_date})"
+underlines = "-~\""
+
+[[tool.towncrier.type]]
+directory = "added"
+name = "Added"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "removed"
+name = "Removed"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "changed"
+name = "Changed"
+showcontent = true
```

Comment:
Should we add a `fixed` ?

Reply:
https://github.com/mozilla-releng/beetmoverscript/pull/126